fip-0026.md

```diff
@@ -103,7 +107,73 @@ Holds information related to the marketplace.
 - bymarketplace
 - byowner
 
-## New actions
+## New actions and end points
+### Get Escrow Listings
+Returns all listings based on status with optional actor parameter for further filtering
+#### /get_escrow_listings
+#### Request
+|Parameter|Required|Format|Definition|
+|---|---|---|---|
+|Status|Yes|Integer Value 1-3| status = 1: on sale, status = 2: Sold, status = 3; Cancelled
+|actor|No|String of actor name|
+|limit|No|Positive Int|Number of records to return. If omitted, all records will be returned. Due to table read timeout, a value of less than 1,000 is recommended.|
+|offset|No|Positive Int|First record from list to return. If omitted, 0 is assumed.|
+##### Example
+```
+{
+    "status": 1,
+    "offset": 0,
+    "limit": 1000,
+    "actor": "wjeo4abnk4c2"
+}
+```
+#### Processing
+* Request is validated per Exception handling
+* All Escrow listings with the status of the provided parameter
+* If `actor` is provided it takes the results and only returns records of that status and with that actor name 
+#### Exception handling
+|Error condition|Trigger|Type|fields:name|fields:value|Error message|
+|---|---|---|---|---|---|
+|Invalid Status value|Value of status is not 1, 2 or 3|400|"status"|Value sent in, i.e. "5"|"Invalid status value"|
+|Invalid limit|limit is not valid|400|"limit"|Value sent in, e.g. "-1"|"Invalid limit"|
+|invalid offset|offset not valid|400|"offset"|Value sent in, e.g. "-1"|"Invalid offset"|
+|No Escrow Listings|There are no escrow listings based on the parameters.|404| | |"No Escrow Listings"|
+#### Response
+|Group|Parameter|Format|Definition|
+|---|---|---|---|
+| |listings|JSON Array|Array of Escrow Listings|
+|listings|id|Int|ID of Escrow Listings|
+|listings|commission_fee|String|commission percentage|
+|listings|date_listed|Int|timestamp when the listing was created|
+|listings|date_updated|Int|timestamp when the listing was last updated|
+|listings|domain|String|the FIO domain being listed|
+|listings|domainhash|String|hash of the FIO domain being listed|
+|listings|owner|String|account name of the owner of the domain|
+|listings|ownerhash|String|hash of the account name of the owner of the domain|
+|listings|sale_price|Int|price the owner wants for the domain, in SUFs|
+|listings|status|Int|Status of the listing.|
+| |more|Int|Number of results remaining|
+##### Example
+```
+{
+  "listings": [
+    {
+        "commission_fee": "6.00000000000000000",
+        "date_listed": 1638229065,
+        "date_updated": 1638229065,
```

Comment:
We should follow convention and return time as "2020-09-11T18:30:56"

Reply:
okay. I just found an example on how to create that date format from the timestamp, so I'll update the C++ code and the FIPs shortly.